CHANGES.rst

```diff
@@ -15,6 +15,7 @@ Bug fixes
 * GH248 Fix an issue causing a ValueError to be raised when using `dask_index_on` on non-integer columns
 * GH255 Fix an issue causing the python interpreter to shut down when reading an
   empty file (see also https://issues.apache.org/jira/browse/ARROW-8142)
+* GH262: Raise an exception when trying to partition on a column with null values to prevent silent data loss
```

Comment:
3.8.1 was already realeased. can you move this, please?

Reply:
rookie mistake